### ISSUE TITLE
move the facts to Slackbot

### ISSUE THREAD

**Issue by afeld** [18F/charlie#288]
[Slack supports custom responses in Slackbot.](https://slack.com/help/articles/202026038-An-introduction-to-Slackbot#add-customized-responses) From [the form where you add them](https://gsa-tts.slack.com/customize/slackbot):

> You can add as many Slackbot responses as you’d like for each input phrase. Put each one on its own line. (A random response will be chosen if there are multiple responses.) 

Moving things there that just pick a random message and respond with a message means we need less custom code.

**Comment by mgwalker** [comment]
Point well-taken. I'm not fundamentally opposed to it, but just to document for history...

The driving force behind moving dog facts to Charlie in the first place is that the text field for Slackbot responses has a maximum length, meaning there is a maximum number of facts that it can hold. We ran into that limit with dog facts and started having to remove some in order to add new ones. We didn't like that, so... we moved it to the bot. Once we had a bot that could spout off facts, we added lots more fact bots, because why not? For added fun, we gave them customizable usernames and avatars.

If the lottery factor comes into play here, though, this is definitely worth keeping a mind on.
